packages/client/src/pages/SignUp/wizards/MentorWizard.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import styled from 'styled-components';
+import Wizard from '../../../components/Wizard';
+import Button from '../../../components/Button';
+
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+import TextField from '@material-ui/core/TextField';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+import { subjects, tags } from '../../../constants.js';
+
+const SignUpChildWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background-color: white;
+    max-width: 400px;
+    min-width: calc(400px - 4em);
+    padding: 2em;
+    color: black;
+`;
+
+const WizardInput = styled.div`
+    margin-bottom: 1em;
+    min-width: 120px;
+`;
+
+const FirstPage = () => {
+
+    const gradeLevelMenuItems = tags.map((item, index) => {
+        return <MenuItem key={index} value={item.id}>{item.value}</MenuItem>;
```

Comment:
should `value` here be `item.value`?

also lets use `key={item.value}` instead of the index so its not order dependent